applications/crossbar/doc/tasks.md

```diff
@@ -376,7 +389,8 @@ curl -v -X GET \
             "node": "whistle_apps@qwd",
             "start_timestamp": 63633924909,
             "status": "executing",
-            "success_count": 50
+            "success_count": 50,
+            "csvs":["in.csv"]
```

Comment:
Explain what this is for?
What happens if this list contains more than one element? none? the same element more than once? a wrong name?

There is already a way to fetch input & output CSVs. `file_name` contains the name of the input CSV.